.golangci.yml

```diff
@@ -5,13 +5,71 @@ linters-settings:
     lines: 80
     statements: 60
 
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
```

Comment:
well, the way to do it is just to pin the version of golangci-lint that you're using.
Which we are doing in c-i (`.github/workflows/lint.yml`).
So a "disable" approach does seem still workable.  The person who wants a new lint version has to change lint.yml and then would be forced to make a decision on things to skip at that point.

I don't have really strong feeling either way.

Reply:
Me neither.  I've seen other projects use the opt-in approach so enable-all doesn't "break" thinks with newer version of golint-ci or their components.   You're quite right about pinning the version though.